tests/python/test_templates_prh.py

```
@@ -35,3 +44,88 @@ def test_javascript_rule_uses_word_boundary_regex_for_js() -> None:
+def prh_rules() -> list[dict]:
+    """templates/prh.yml をロードして rules リストを返す．
+
+    fail-fast 設計：YAML が空・rules キー欠落・rules が list でない いずれの
+    異常でも明示的な例外を送出する．silent fallback で「rules が空のリスト」
+    として扱うと回帰テストが偽 green になるため避ける．
+    scope=module で 12 件のパラメタライズドテストにわたり 1 度だけロードする．
+    """
+    with _PRH_PATH.open(encoding="utf-8") as f:
+        data = yaml.safe_load(f)
+    if data is None:
+        raise ValueError(f"{_PRH_PATH} が空または無効な YAML です")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)